locate/src/config/constants.js

```diff
@@ -1,3 +1,19 @@
+const constants = {
+  RUN_LOCATE_MODEL:
+    //"https://locate-bknd-service-dot-airqo-250220.uc.r.appspot.com/api/v1/map/parishes",
+    "http://localhost:4001/api/v1/map/parishes",
```

Comment:
@lilly-muyama , we can move http://localhost:4001/api/v1/ ... urls to devConfig and https://locate-bknd-service-dot-airqo-250220.uc.r.appspot.com/api/v1/ to prodConfig. It helps when running in different environments.